contract_compassion/wizard/end_contract_wizard.py

```diff
@@ -65,6 +66,12 @@ def end_contract(self):
         self.ensure_one()
         contract = self.contract_id
 
+        # if not self.has_new_hold:
+        """ Warning maybe to change when R4 will be stable...
+        # Currently Connect refuse sponsorship cancel without
+        expiration hold date"""

```

Comment:
This won't change apparently. So the behaviour is to send the current datetime and ignore the hold id returned that will expire very quickly. Here is the answer from the U.S.:
```
When you cancel a commitment, the beneficiary must be put on hold, at least momentarily. This was because most Partners keep the beneficiary on Sponsor Cancel Hold or No Money Hold for some time to try and maintain the sponsorship.

If you don’t wish to use this, set the hold expiration date to the current time. You’ll receive a Hold ID but you should be able to ignore it.
```